osmoutils/osmocli/dynamic.go

```diff
@@ -0,0 +1,48 @@
+package osmocli
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/gogo/protobuf/proto"
+
+	"github.com/osmosis-labs/osmosis/osmoutils"
+)
+
+type Descriptor interface {
+	GetCustomFlagOverrides() map[string]string
+	AttachToUse(str string)
+}
+
+// fields that are not provided as arguments
+var nonAttachableFields []string = []string{"sender", "pagination", "owner", "admin"}
```

Comment:
to reviewers: are there some other fields which should not be passed as arguments of cli commands?